engines/xs/predict-url.js

```diff
@@ -16,19 +16,19 @@
 const predictFileName = (os) => {
 	switch (os) {
 		case 'mac64': {
-			return 'mac';
+			return 'mac64';
 		}
-		case 'linux32': {
-			return 'lin32';
+		case 'mac64arm':{
+			return 'mac64arm';
 		}
 		case 'linux64': {
 			return 'lin64';
 		}
-		case 'win32': {
-			return 'win';
+		case 'linux64arm': {
```

Comment:
Each `case` must be one of the options in the README, i.e. either `mac64`, `mac64arm`, `win32`, `win64`, `linux32, or `linux64`.